src/tests/test_users.py

```diff
@@ -102,24 +110,42 @@ def test_fetch_users(test_app, monkeypatch):
     assert response.json() == [{k: v for k, v in entry.items() if k != "access_id"} for entry in mock_user_table]
 
 
-def test_create_user(test_app, monkeypatch):
+@pytest.mark.asyncio
+async def test_create_user(test_app_asyncio, test_db, monkeypatch):
 
     # Sterilize DB interactions
-    mock_user_table = deepcopy(USER_TABLE)
+    mock_user_table = deepcopy(USER_TABLE_FOR_DB)
     mock_access_table = deepcopy(ACCESS_TABLE)
-    _patch_session(monkeypatch, mock_user_table, mock_access_table)
+    monkeypatch.setattr(crud, "database", test_db)
+    monkeypatch.setattr(security, "hash_password", pytest.mock_hash_password)
+    await populate_db(test_db, db.accesses, ACCESS_TABLE)
+    await populate_db(test_db, db.users, USER_TABLE_FOR_DB)
```

Comment:
Correct me if I'm wrong, but since this inserts data and doesn't clean beforehand, conducting test sequentially will result in some tests starting with an already filled db?

Reply:
`flush_test_db`  is called at the end of every test when the fixture `test_db` is no longer used. This function keeps the db schema but delete all rows.